### PR TITLE
[ADD] add fields to the product view

### PR DESCRIPTION
**Pull request by victor-champonnois**
Add two non-relational fields to the foodhub product model and add them to the product view

task : https://gestion.coopiteasy.be/web#id=7559&view_type=form&model=project.task

@robinkeunen @huguesdk @remytms

### CODE REVIEW

**Comment by victor-champonnois** [comment]
Fixup : running pre-commit
Je me rends compte que pre-commit à aussi modifié un fichier de provelo. Oups ! Je l'enlève ?

File: foodhub_custom/models/foodhub_product.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from odoo import fields, models
```
Comment (robinkeunen) [inline foodhub_custom/models/foodhub_product.py]: We usually name the files following the model name. Here, that would be `product.py`.

File: foodhub_custom/models/foodhub_product.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+
+class FoodhubProduct(models.Model):
+    _inherit = "product.template"
+    location = fields.Char("Location")
```
Comment (robinkeunen) [inline foodhub_custom/models/foodhub_product.py]: I like to use named parameters for all fields parameters. Here, for example, I would write :

```py
location = fields.Char(string="Location")
# or
location = fields.Char()
```

Whats does @remytms and @huguesdk think? What are your tastes ?

**Review by robinkeunen — CHANGES_REQUESTED** [review]
Good job, just a few refactoring requests.

File: foodhub_custom/models/foodhub_product.py (hunk 1 above)
Comment (victor-champonnois) [reply foodhub_custom/models/foodhub_product.py]: Done !

File: foodhub_custom/models/foodhub_product.py (hunk 2 above)
Comment (victor-champonnois) [reply foodhub_custom/models/foodhub_product.py]: OK, for now I used the "string=Location" form.

**Comment by victor-champonnois** [comment]
Thank you @robinkeunen, I made the requested changes.

**Review by robinkeunen — APPROVED** [review]
LGTM 👏

Do not forget to squash your commits.

**Comment by victor-champonnois** [comment]
Just removed the accidental changes to provelo

**Comment by victor-champonnois** [comment]
Small fix !

**Review by robinkeunen — APPROVED** [review]

**Comment by carmenbianca** [comment]
Added two (one-liner) commits. They should probably be squashed. The dependency is added because there already already was a producer field in use (i.e., contained data) in the DB, but nothing depended on the module that provides this field.

I asked Pol whether `website_sale_detailed_product_description` was manually added. Also let me know whether this module is the correct place to add this dependency.

**Review by robinkeunen — CHANGES_REQUESTED** [review]
The analysts do install modules themselves, we do not generally install them through dependencies. Since you now rely on the module for a field, you must add it to your dependencies as you did.

Indeed, a single commit for this change would make more sense.

Congrats on the first commit 👏

**Comment by carmenbianca** [comment]
Pol has put this on hold for now while clarifying with customer.

**Comment by carmenbianca** [comment]
Moved @coccinouille 's code to a new module.

**Pull request merged**